src/models/nftRewardTier.ts

```diff
@@ -3,6 +3,11 @@ import { CurrencyOption } from './currencyOption'
 
 export const NFT_METADATA_CONTRIBUTION_FLOOR_ATTRIBUTES_INDEX = 0
 
+// eslint-disable-next-line
```

Comment:
@johnnyd-eth whats teh rule being disabled here? shouldnt disable rules if we can help it

Reply:
Can't have underscores in `type`s. Not sure how else to name V1.1

Reply:
gotit